packages/js/product-editor/src/blocks/search-visibility/types.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import { BlockAttributes } from '@wordpress/blocks';
+
+export interface SearchVisibilityBlockAttributes extends BlockAttributes {
+	label: string;
+}
+
+export type ProductCatalogVisibility =
+	| 'visible'
+	| 'catalog'
+	| 'search'
+	| 'hidden';
```

Comment:
Would it make sense to just have a re-usable type that both blocks use, instead of defining this in both types?
This type should really be part of the `data` package, exported from the `products` store, similar to ProductStatus here: https://github.com/woocommerce/woocommerce/blob/e3194c25e340e1f9d5b5d0da29eae4ec51eb726c/packages/js/data/src/products/types.ts#L13